package.json

```diff
@@ -21,6 +21,7 @@
     }
   ],
   "dependencies": {
+    "es6-shim": "^0.26.1",
```

Comment:
@nathansobo: instead of including a single polyfill, I opted to use a more comprehensive library. Hopefully, this will be useful for the next features and we don't have to include anything else :smile: